tests/test_chat_trigger.sh

```
@@ -0,0 +1,349 @@
+else
+  fail "anthropics/claude-code-action が SHA pin されていない"
+fi
+
+# スレッド履歴取得ステップ
+if grep -F "thread_history" "$CHAT_WORKFLOW" > /dev/null; then
+  pass "thread_history ステップが存在する"
+else
+  fail "thread_history ステップが存在しない"
+fi
+
+if grep -F 'gh api "repos/${REPO}/issues/${ISSUE_NUMBER}/comments" --paginate' "$CHAT_WORKFLOW" > /dev/null; then
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)